node/t3rn-parachain/src/chain_spec.rs

```diff
@@ -166,8 +205,9 @@ pub fn polkadot_config() -> ChainSpec {
         "t3rn_polkadot",
         ChainType::Live,
         move || {
+            // TODO: needs updating
             polkadot_genesis(
-                // Invulnerable collators
+                // Invulnerable collators FIXME: these are NOT the right collators
```

Comment:
We need to figure out the strategy on what we do here, maybe just pregen the keys and update this once we have. Then we can prefund the collators with the bond at genesis.